README.md

```diff
@@ -1,101 +1,96 @@
-# true[X]  Web Integrations
-Documentation and reference apps for integrating true[X]'s CTV web ad renderer
+# TruexAdRenderer Web Integration: Quick Guide
 
+This document describes the how to use Infillion's TrueX web ad renderer, a library that allows the presentation of
+interactive ads, intended for use in "Full Experience Playback" (FEP) scenarios such as video web sites or applications.
 
-## Setup
-This document describes the initial steps needed to make use of the `TruexAdRenderer` in an HTML5 web application intended for Smart TVs and game consoles, i.e. for the so-called "10 foot" experience.
+The library provides a class called the `TruexAdRenderer` (TAR). TAR is supports running within any web content, 
+specifically for desktop and mobile web content, as well as for the "Connected TV" (CTV) platforms, such as Smart TVs
+and game consoles, i.e. for the so-called "10 foot" experience.
 
-The true[X] ad renderer is available as an `npm` module. For the typical web app based around a `package.json` project file, one adds the true[X] dependency as follows:
+## Setup
+The TrueX ad renderer is available as an `npm` module. For the typical web based development around a `package.json` 
+project file, one adds the TrueX dependency as follows:
 ```sh
-npm add @truex/ctv-ad-renderer
+npm add @truex/ad-renderer
 ```
+this will add an entry in the `"dependencies"` section in the `package.json` file, something like:
+```json
+    "dependencies": {
+        "@truex/ad-renderer": "1.11.0",
+```
+You then build and run your web app like usual, e.g. invoking `npm start` for webpack-based projects.
 
-## Next Steps
-
-[TruexAdRenderer-Web Documentation](DOCS.md) outlines use of the ad renderer, including its API interface and the observable events it fires
+### Code Sample
 
-### Reference Apps
+To actually integrate the TrueX ad renderer, you have to create and invoke it during your app's video playback 
+when the time of an ad break is reached.
 
-1. [Generic Reference App](https://github.com/socialvibe/truex-ctv-web-reference-app)
-1. [Google Ad Manager: Server-side Ad Insertion Reference App](https://github.com/socialvibe/truex-ctv-web-google-ad-manager-reference-app)
-1. [Google Ad Manager: Client-side Ad Insertion Reference App](https://github.com/socialvibe/truex-ctv-google-ima-csai-ref-app)
-
-### Code Sample
+The following code provides an example of the style of how to integration to TAR, once a Truex ad has been detected 
```

Comment:
I don't understand this sentence

Reply:
How about:

Once a TrueX ad is detected, the renderer needs to be created and displayed. The following code provides an example of the typical approach of integrating to TAR.

Reply:
fixed